index.js

```diff
@@ -37,17 +37,29 @@ function eyesWithout(fn) {
 function eyesWith(fn) {
   return function () {
     var windowSize = eyes.defaultWindowSize;
-    if (isPassedWindowSizeArgument(arguments)) {
-      windowSize = arguments[2];
+    var specVersion = '';
+
+    if (isPassedParameterArgument(arguments)) {
```

Comment:
I think that a nicer approach will be destructering args, something like this:
https://github.com/wix-incubator/puppeteer-eyes.it/blob/master/lib/eyes-with.js#L4

Reply:
agree, but I don't think it's supported here. it's not es6 project as far as i understand

Reply:
it is node... you can also update .nvmrc to use latest node